src/main/java/core/basesyntax/figure/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.figure;
+
+import base.Figure;
+
+public class Circle extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the class definition.